react/features/shared-video/components/web/SharedVideoDialog.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import { FieldTextStateless } from '@atlaskit/field-text';
+import React from 'react';
+
+import { defaultSharedVideoLink } from '../../';
+import { Dialog } from '../../../base/dialog';
+import { translate } from '../../../base/i18n';
+import { connect } from '../../../base/redux';
+import AbstractSharedVideoDialog from '../AbstractSharedVideoDialog';
+
+/**
+ * Component that renders the video share dialog.
+ *
+ * @returns {React$Element<any>}
+ */
+class SharedVideoDialog extends AbstractSharedVideoDialog<*> {
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        const { t, value } = this.props,
+            { okDisabled } = this.state;
+
+        return (
+            <Dialog
+                hideCancelButton = { false }
+                okDisabled = { okDisabled }
+                okKey = { t('dialog.Share') }
+                onSubmit = { this._onSetVideoLink }
+                titleKey = { t('dialog.shareVideoTitle') }
+                width = { 'small' }>
+                <FieldTextStateless
```

Comment:
Set the `name` prop to the previous name so the torture tests pass.

Reply:
All the review remarks have been implemented @saghul. Thank you!